common/remote/sockfile.cpp

```diff
@@ -6227,6 +6241,7 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
 
         int cursorHandle = requestTree->getPropInt("handle");
         OutputFormat outputFormat = outFmt_Xml;
+        unsigned __int64 replyLimit = 0;
```

Comment:
does it make sense to initialize to the default value here as well?

Reply:
yes I think so, it's clearer, the compiler will likely optimize it away anyway. And makes sure all code paths are initialized even if current ones where not used don't matter.

Reply:
Ah you meant to : requestTree->getPropInt64("replyLimit", defaultDaFSReplyLimitKB) * 1024

Could do, and if zero cost perhaps, but as no practical difference and there's extra expense of going through another call to getPropInt virtual etc, on balance better as it is.

Reply:
Yes, I think one of my comments didn't go through, I think the chooseN is initialized the same way. 
Anyway,the only extra expense I see in this case would be the second "* 1024". Not a big deal